Composer/packages/client/src/utils/dialogUtil.ts

```diff
@@ -73,23 +82,52 @@ export function insert(content, path: string, position: number | undefined, data
     $type: data.$type,
     ...seedNewDialog(data.$type, {}, optionalAttributes),
   };
+  return newStep;
+}
 
-  const insertAt = typeof position === 'undefined' ? current.length : position;
-
-  current.splice(insertAt, 0, newStep);
+export function generateRegexExpression(intent: string, pattern: string) {
+  return { intent, pattern };
+}
 
-  set(content, path, current);
+export function createNewTrigger(dialog: DialogInfo, data: TriggerFormData): DialogInfo {
```

Comment:
The return type `DialogInfo` I believe is an indexed dialog type, but here you didn't
 reindex it, and seems no need to reindex it. So I would suggest return `content: string`.